content/en/docs/best-practices/channel-naming.md

```diff
@@ -27,13 +27,13 @@ is a diagram that shows the relationship of operator versions to channels:
 
 In the diagram above you can see the following:
```

Comment:
the [image](https://github.com/operator-framework/olm-docs/blob/67fa35f01d9584c9702f4b4c717db53dafcaf708/content/en/docs/best-practices/images/channel-naming1.png)s were not updated accordingly